hcn/hcn.go

```diff
@@ -117,6 +127,13 @@ func V2ApiSupported() error {
 	return platformDoesNotSupportError("V2 Api/Schema")
 }
 
+func V2SchemaVersion() SchemaVersion {
```

Comment:
Do you really mean 2.1? You should be able to use `schemaversion.SchemaV21()` here.

Reply:
This is hns v2 schema :) . We have our own schema versioning.